src/auth/AxiosJwtAuthService.js

```diff
@@ -59,24 +62,41 @@ class AxiosJwtAuthService {
     this.csrfTokenService = new AxiosCsrfTokenService(this.config.CSRF_TOKEN_API_PATH);
     this.authenticatedHttpClient = this.addAuthenticationToHttpClient(axios.create());
     this.httpClient = axios.create();
+    configureCache().then((cachedAxiosClient) => {
```

Comment:
I'm wondering if we should add a try/catch around this call to configureCache... if it fails for some reason, we can still proceed with loading, we just might not be able to cache requests.  I'm thinking that perhaps we set the `cachedAuthenticatedHttpClient` and `cachedHttpClient` to the _non-cached_ versions and log an error to new relic via `logError`.

It seems like it might be a good idea, given that `configureCache` is interacting with a bunch of browser APIs which may or may not exist.